Packages/com.unity.multiplayer.samples.coop/Utilities/SceneManagement/SceneLoader.cs

```diff
@@ -0,0 +1,141 @@
+using System.Collections.Generic;
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Unity.Multiplayer.Samples.Utilities
+{
+    public class SceneLoader : NetworkBehaviour
+    {
+        /// <summary>
+        /// This singleton is a utility handling the loading of scenes. It loads scene using the SceneManager, or, on
```

Comment:
```suggestion
        /// Adds loading screens by wrapping scene management APIs. It loads scene using the SceneManager, or, on
```
Just to put the emphasize on the right thing. Users with no loading screens don't need this class, correct?